crypto/crypto.go

```diff
@@ -13,8 +13,12 @@ import (
 )
 
 // SecureToken creates a new random token
-func SecureToken() string {
-	b := make([]byte, 16)
+func SecureToken(options ...int) string {
```

Comment:
why do we take in a variable argument list here?

Reply:
Oh this is a way of [taking in optional arguments](https://stackoverflow.com/questions/2032149/optional-parameters-in-go). `SecureToken()` is used to generate recovery codes which are of length 8 instead of the default of length 16, passing in an int (e.g. SecureToken(8)) will return a  length 8 random string